### PR TITLE
Artist updater

### LINKED ISSUE #56: Description
Artists description through Wikipedia scrapping

### LINKED ISSUE #57: Tokens swap

### LINKED ISSUE #58: ArtistsDB population solution
1. db population script
2. artist Updater
3. New Fields for Artist and Event
4. Refactoring issues

### LINKED ISSUE #59: Artist Iterator
first version of simple async iteration over artist keywords

### PR DESCRIPTION
**Pull request by Fanter99**

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -16,6 +18,12 @@ DB_IP="localhost"
 DB_PORT="5432"
 DB_NAME="postgres"
 
+ARTISTS_DB_LOGIN="artists_postgres"
```
Comment (Forsee41) [inline .env-example]: remove this db, use main one instead

File: band_tracker/updater/ApiClient.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import logging
```
Comment (Forsee41) [inline band_tracker/updater/ApiClient.py]: use snake case module naming

File: tests/test_data/artists/gosha.json [hunk 3]
```diff
@@ -3,7 +3,8 @@
 	"socials": {
 		"instagram": "https://gosha_inst.com",
 		"youtube": "https://gosha_youtube.com",
-		"spotify": "https://gosha_spotify.com"
+		"spotify": "https://gosha_spotify.com",
+		"wiki": "https://en.wikipedia.org/wiki/Gosha_woman"
```
Comment (Forsee41) [inline tests/test_data/artists/gosha.json:7]: That is very insulting and inappropriate, I like it

File: scripts/artists_DbPopulation.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+            await connection.execute(update_query, checked_ids)
+
+
+async def init_db(db_pool: Pool) -> None:
+    query = "UPDATE artists SET checked = false;"
+    await db_pool.execute(query)
+
+
+async def main() -> None:
```
Comment (Forsee41) [inline scripts/artists_DbPopulation.py]: This might need some refactoring also

**Review by Forsee41 — CHANGES_REQUESTED** [review]

File: tests/test_data/artists/gosha.json (hunk 3 above)
Comment (Fanter99) [reply tests/test_data/artists/gosha.json:7]: that's the real wikipedia page though

**Review by Forsee41 — APPROVED** [review]

**Pull request merged**